thirdparty/README.md

```diff
@@ -0,0 +1,40 @@
+# Third-party libraries
+
+Unchanged third-party libraries included in the add-on.
```

Comment:
```suggestion
Third-party libraries included in the add-on.
```
Unlike for the current master the *unchanged* part is not true here for all files.

Reply:
You're right, I've changed that